behave_django/runner.py

```diff
@@ -0,0 +1,62 @@
+try:
+    from django.test.runner import DiscoverRunner
+except ImportError:
+    from django.test.simple import DjangoTestSuiteRunner as DiscoverRunner
+
+from django.core.management import call_command
+
+try:
+    from django.shortcuts import resolve_url
+except ImportError:
+    # support Django 1.4, which has no resolve_url() yet
+    from django.shortcuts import redirect
+
+    resolve_url = lambda to, *args, **kwargs: \
+        redirect(to, *args, **kwargs)['Location']
+
+from behave_django.testcase import (BehaviorDrivenTestCase,
+                                    ExistingDatabaseTestCase)
+
+
+class BehaviorDrivenTestRunner(DiscoverRunner):
+    testcase_class = BehaviorDrivenTestCase
+
+    def before_scenario(self, context):
```

Comment:
I think a mixin would be better. `BehaveHooksMixin`. With mixin, the inheritance would be more defined.

Right now, `ExistingDatabaseTestRunner` inherits from `BehaviorDrivenTestRunner`, which then inherits from `DiscoverRunner`. I think it's more elegant if they both inherited from `DiscoverRunner`, and had a mixin for the `before_scenario` methods. :grinning: